STM32/DC/Core/Src/DCBoard.c

```diff
@@ -285,24 +366,49 @@ static void checkButtonPress()
     }
 }
 
-// Public member functions.
+/***************************************************************************************************
+** PUBLIC FUNCTIONS
+***************************************************************************************************/
+
+/*!
+** @brief Setup function called at startup
+**
+** Checks the hardware matches this FW version, starts the USB communication and starts the ADC. 
+** Printing is synchronised with ADC, so it must be started in order to print anything over the USB
+** link
+*/
 void DCBoardInit(ADC_HandleTypeDef *_hadc, I2C_HandleTypeDef *_hi2c, WWDG_HandleTypeDef* hwwdg)
 {
-    BoardType board;
-    if (getBoardInfo(&board, NULL) || board != DC_Board)
-        return;
+    setFirmwareBoardType(DC_Board);
```

Comment:
Where are these functions defined and what is their purpose? Haven't seen them before and can't find the implementation in the PR.

Reply:
These functions tell systemInfo what board type and oldest supported PCB type the firmware was compiled for.